DESCRIPTION

```diff
@@ -90,7 +90,8 @@ Imports:
     tools,
     utils,
     uuid (>= 0.1-4),
-    xml2 (>= 1.3.0)
+    xml2 (>= 1.3.0),
+    curl
```

Comment:
Drop this for now. We may apply the internet connectivity check to the entire code base later. But for now let's avoid adding another dependency, even though it is imported by httr.